VS_Code/readme.md

```diff
@@ -0,0 +1,102 @@
+
+## Prerequisites
+
+* Java SE 8, Java SE 11 or Java SE 17 is required.
+* Payara Server 5 is required. You can download Payara Server 5 from https://www.payara.fish/downloads/ 
+* The VS Code IDE is required. You can download the VS Code IDE from https://code.visualstudio.com/download
+
+## Download
+
+Download the source code zip file and expand it somewhere in your file system. Note that this is a Maven project. https://github.com/eclipse-ee4j/cargotracker/archive/master.zip
+
+## VS Code IDE Set-up
+
+### Installing Necessary Tools
+
+Start the VS Code IDE
+
+Go to ->Extensions (Ctrl+Shift+ X)
+
+Search for [Payara Tools](https://marketplace.visualstudio.com/items?itemName=Payara.payara-vscode) and [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack) and install Payara Tools and Extension Pack for Java.
+
+ ![Payara_Tools](<images/payara_tools.PNG>)
+
+ ![Extension_Pack_for_Java](<images/extension_pack_for_java.PNG>)
+
+### Adding Payara Server
+
+Once the Payara Tools plugin is installed, 
+1. Click on the Payara icon on the left side bar below the Extensions icon,then click on the '+' icon on the Servers.
+
+![Adding_Server](<images/adding_payara_server_1.PNG>)
+
+* Before doing the next step,please make sure that you have properly set up JAVA_HOME on your PC.
+2. choose local domain -> browse the Payara server,then select the directory where you installed the Payara 5. 
+
+![Local_Domain](<images/Adding_payara_server_2.png>)
+
+
+![browse](<images/Adding_payara_server_3.png>)
+
+3. Name it Payara5 and hit next. Select default domain1.
+
+![Payara5](<images/Adding_payara_server_4.png>)
+
+![Local_domain1](<images/Adding_payara_server_5.png>)
+
+4. Now ,a payara5 name icon with the Payara logo comes under the servers.Right-click on that icon,and select start,it will automatically start the Payara 5.
+
+![running](<images/Running_payara_server_1.png>)
+
+5. A small green color triangle comes under the Payara5 logo in the servers,this means your Payara server has been successfully started.You can verify if you wanted by visiting localhost:8080
+
+![running_sucessful](<images/Running_payara_server_2.png>)
+
+### Importing Code
+
+1. Go to File -> Open Folder ->Select the root directory of the Cargo Tracker project in your file system, and Hit finish.VS Code will automatically identify it as a Maven project.
+
+![import](<images/importing_cargotracker_1.png>)
+
+2. This will take a few minutes to import the project for the first time.
+
+![import2](<images/importing_cargotracker_2.png>)
+
+![import3](<images/importing_cargotracker_3.png>)
+
+### Building and Running on Payara 5
+1. After the project loads,Go under the maven tab on the left bottom side,the eclipse cargo tracker name will be shown.Right-click on it and proceed with run maven commands clean,package,install and compile(which will take a while the very first time as Maven downloads dependencies)
```

Comment:
The exact Maven goals that need to be executed are clean and package. Please fix this.